lib/packwerk/loader.rb

```diff
@@ -16,7 +16,7 @@ def autoloaders
     def root_dirs
       return super if Zeitwerk::VERSION < ROOT_DIRS_DEPRECATION_VERSION
 
-      __getobj__.__roots
+      __getobj__.dirs(namespaces: true)
```

Comment:
Here's documentation on this public API: https://github.com/fxn/zeitwerk#introspection

Reply:
Good call @jeffmeyers , changed!

Reply:
Since this is public API I would refactor this to be the super case / method we call into on the facade class.
```suggestion
      __getobj__.root_dirs
```

Reply:
I think that's a great suggestion, if I correctly understood what you meant! Check out the commit I pushed up and let me know if it was what you had in mind.